src/bases.d.ts

```diff
@@ -171,6 +171,11 @@ export interface StatefulMixin<S extends State>{
 	readonly state: S;
 
 	/**
+	 * A readonly reference to the stateFrom provided to the widget on instantiation.
+	 */
+	readonly stateFrom?: StoreObservablePatchable<S>;
```

Comment:
it seems I was able to implement the `getter` in `createStateful` using `?`, is the suggested way better practice?